Update documentation about BYOI RAID Capabilities (#5577)
ovh/docs: pages/cloud/dedicated/bring-your-own-image/guide.en-gb.md

If your server has a **uefi** boot type, be sure to add an **EFI** partition to your image template.

**About RAID**

> There is no possibility right now to handle software RAID at the install.

Reviewer: ```suggestion
Bring Your Own Image does not support software RAID configuration at install-time.
```

> You can always configure it later, if your image was from a RAID (mdadm), in post-installation.

Reviewer: ```suggestion
You can still configure it later, in a post-installation script, for instance if your image contains md RAID 1 arrays.
```

> Hardware RAID is supported, if you have the right hardware, because it is configured before the image is deployed on disk.

Reviewer: ```suggestion
Hardware RAID is supported, if your server supports it, because it is configured before the image is deployed on disk.
```
